packages/mdc-tab/foundation.js

```diff
@@ -102,7 +110,9 @@ class MDCTabFoundation extends MDCFoundation {
     this.adapter_.setAttr(strings.ARIA_SELECTED, 'true');
     this.adapter_.setAttr(strings.TABINDEX, '0');
     this.adapter_.activateIndicator(previousIndicatorClientRect);
-    this.adapter_.focus();
+    if (this.focusOnActivate_) {
```

Comment:
I'd be tempted to make this a parameter for the `activate` function that can be passed by the `TabBar`. Then, `focusOnActivate` won't need to be stored in each `Tab` instance.

Reply:
So `TabBar` would pass it's value for `focusOnActivate` to a `Tab` via the `activate` method. I realize this makes more changes to the `Tab` foundation interface. If you're trying to prevent breaking changes, we can roll with the existing solution. Let me know what you think.

Reply:
Your idea is the first idea I had, but I thought against it for a couple of reasons:

* It would make the `activate` API more awkward, given that it already has an optional `ClientRect` parameter
* Semantically, it seems unlikely that you'd want to call this with different values across the same instance's lifetime; you'd probably just want to set the behavior once and roll with it

It wouldn't necessarily be a breaking change that way, since it could also be optional with a default.

Reply:
True true, those are all good points.